pkg/template/helm_template.go

```diff
@@ -167,3 +178,49 @@ func (t *HelmTemplate) writeFile(dstPath, subPath string, content []byte) (strin
 
 	return newPath, nil
 }
+
+type helmTemplateCMDArgs struct {
```

Comment:
Will rename this from `helmTemplateCMDArgs` to `helmTemplateCmdArgs` to be consistent with camelcase naming.